app/assets/stylesheets/font-awesome/_animated.scss

```diff
@@ -1,36 +1,34 @@
-@if $fa-enable-utilities {
```

Comment:
Hi @hello-jason Thanks for your contribution :)
Please keep the if statement in order to don't break the $fa-enable-utilities configuration option to not include utilities classes.